src/test/java/com/radixdlt/crypto/HashTest.java

```diff
@@ -60,7 +60,7 @@ public void verify_that_random_is_not_null() {
 
 	@Test(expected = IllegalArgumentException.class)
     public void verify_that_an_error_is_thrown_for_too_short_hex_string_constructor() {
-		new Hash("deadbeef");
+		assertNotNull(new Hash("deadbeef"));
```

Comment:
how can this be non null? The constructor will throw an exception, how can it return an non-null instance of `Hash` if an exception was thrown?

Reply:
It can't be null, but calling a constructor without using the constructed object is a compiler warning.

Reply:
Per previous comments, used to suppress compiler warning, but done differently now.